plugins/hydra_optuna_sweeper/hydra_plugins/hydra_optuna_sweeper/config.py

```diff
@@ -173,6 +173,9 @@ class OptunaSweeperConf:
     # Number of parallel workers
     n_jobs: int = 2
 
+    # Maximum authorized failure rate for a batch of parameters
+    max_failure_rate: float = 0.0
```

Comment:
Hi @tmct,
Note that it's a 'rate' so specifying, 1.0 would mean that there is no 'maximum' as all trials are allowed to fail. Similarily, specifying 0.0 would result in stopping if any error is encountered.

Does this cover what you want, or do you still mean something else?

Reply:
The inequality check `if len(failures) / len(returns) > self.max_failure_rate` is a *strict* inequality. Since `len(failures) / len(returns)` should never be larger than `1.0`, I think the use-case described by @tcmt is covered.

Reply:
Ah yes, apologies - I had not spotted it was a strict inequality. These sound like the right behaviours for `0.0` and `1.0`. Many thanks!